monitor/app/staking/metrics.go

```diff
@@ -0,0 +1,36 @@
+package staking
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	rewardsAvg = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "monitor",
+		Subsystem: "staking",
+		Name:      "avg_rewards",
```

Comment:
suggest: `monitor_staking_rewards_avg_percentage`, this makes it easier to understand when searching tthrough metrics in grafana